src/shared/components/section/section.css

```diff
@@ -15,6 +15,15 @@
   background-color: #FFF;
 }
 
+.condensed {
+  min-height: 0;
+}
+
+.whiteCondensed {
+  composes: white;
```

Comment:
woah - never knew about this! thanks for showing me something new.

Reply:
Yeah, there's a couple other places we've been using `composes` w/ CSS Modules in the app - it's a nice way of composing multiple classes together to create a new one, so you only need to apply one CSS class to an element instead of multiple ones (which is sometimes seen w/ patterns like BEM).